.rubocop.yml

```diff
@@ -274,6 +302,11 @@ Naming/PredicateName:
   Exclude:
     - spec/**/*
 
+Naming/VariableNumber:
+  Description: 'Use the configured style when numbering symbols, methods and variables.'
+  StyleGuide: '#snake-case-symbols-methods-vars-with-numbers'
+  Enabled: false
+
```

Comment:
This is enabled automatically and I'm disabling it. It's a bit picky. You either have to enforce all variables to be in the style of `variable1` or `variable_1` with the underscore. I don't have a preference either way and can live with both styles :D